README.md

```diff
@@ -46,73 +71,57 @@ This will output similar to the following:
-* `transcribe-json`: JSON Data received from an API request needs to be modified to be displayed to the end user, and some of these actions are intensive. This shows the duration it took to completly modify and return the JSON data.
-* `cache-check`: Local or Remote or Both types of caching are implemented on this endpoint. This shows the duration it took to check with said caches for the relevant data.
+Of the above output, the only field that you should worry about is `duration`, as this tells you in milliseconds how long whatever action is related took. Of the actions that might be shown below is an explanation of what they mean:
+  * `api-request`: An API Request was needed to be made to the backend server, and this shows the duration that the request took.
+  * `transcribe-json`: JSON Data received from an API request needs to be modified to be displayed by the website. This shows the duration the modification took.
+  * `cache-check`: Local or Remote or Both types of caching are implemented on this endpoint. This shows the duration it took to check with said caches for the relevant data.
 
 Currently `loadStats()` is unable to display how long certain actions took, which are listed below:
+  * The time `ExpressJS` took to handle the request itself.
+  * The time `EJS` templates took to handle creating and displaying the page.
 
-* The time `ExpressJS` took to handle the request itself.
-* The time `PugJS`/`Jade` templates took to handle creating and displaying the page.
+## Microservices
 
----
-
-## Developers
-
-To run and test `package-frontend` locally should be rather simple.
-
-After cloning the repository, run `npm install` within the root of the folder.
-
-Then copy or rename `app.example.yaml` to `app.yaml`. When working with a local app config file you can change settings of the backend API to interact with to use a local version, or change the port the frontend is exposed on.
-
-Within the config file exists the key `GOOGLE_APPLICATION_CREDENTIALS` this key is used to point to a KeyFile containing Google Application Credentails to interact with Remote Caches. If you don't have access to this file set it to `"no-file"` like so:
-
-```yaml
-GOOGLE_APPLICATION_CREDENTIALS: "no-file"
-```
+Some tasks within the frontend are run in their own dedicated microservice. To read more about why some aspects are moved to a microservice and how they behave read [here](./microservices/README.md).
 
-This is a special value that will cause the caching service to automatically ignore any cache requests, and force request through to the backend service. Allowing to fail gracefully.
+### Social Image Cards
 
-### Running In Dev Mode
+Social Image Cards allow the creation of dynamic Package Cards to be displayed, either when linked to directly, or when a link to a packages page on the frontend has been shared to services that display the image websites prompt to.
```

Comment:
```suggestion
Social Image Cards allow the creation of dynamic Package Cards to be displayed, either when linked to directly, or when a link to a packages page on the frontend has been shared to services that display the image website's prompt to.
```
I'm not quite sure I understand the last bit of this? "display the image websites prompt to", it might well make sense in context but I can't quite work it out.

Reply:
I had trouble figuring out how to say this one. Essentially the `meta` tags of a website can prompt using `twitter` meta tags or `og` meta tags to link an image when displaying a pages rich content when shared. Which those meta tags point to the social image cards.

So I wasn't quite sure how to quickly say that block of text here.

Reply:
Right i think i get you, sort of? What exactly is meant by "image website"? Maybe something like:
> or when a link to a Pulsar package page is shared to a service that displays the image when... (and this is where I'm not actually sure I quite get it or at least how to explain accurately.